PhotoMap/Features/Map/Coordinators/MapCoordinator.swift

```diff
@@ -9,7 +9,12 @@ import UIKit
 import Combine
 import CoreLocation
 
-class MapCoordinator: Coordinator {
+protocol ImagePickerProtocol {
+    var imagePickerSourceSubject: PassthroughSubject<UIImagePickerController.SourceType, Never> { get }
+    var showImagePickerSubject: PassthroughSubject<UIImagePickerController, Never> { get }
+}
```

Comment:
If it can be used outside MapCoordinator than better to extract to a separated file